src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const sevenOff = 50;
+  const threeOff = 20;
```

Comment:
bad variable names, don't attach them to days, use more abstract entities like `small` `large`